doorstop/core/publisher.py

```diff
@@ -154,9 +154,8 @@ def _lines_index(filenames, charset='UTF-8', tree=None):
         '<meta http-equiv="content-type" content="text/html; '
         'charset={charset}">'.format(charset=charset)
     )
-    yield '<style type="text/css">'
-    yield from _lines_css()
-    yield '</style>'
+    yield '<link rel="stylesheet" href="assets/doorstop/bootstrap.min.css" />'
+    yield '<link rel="stylesheet" href="assets/doorstop/general.css" />'
```

Comment:
```suggestion
    yield '<link rel="stylesheet" href="assets/doorstop/general.css" />'
    yield '<title>Doorstop Index</title>'
```

Right now the Index page has no Title. Give it a title.